lib/rubygems/await.rb

```diff
@@ -262,7 +262,11 @@ def process_element(name, tuples)
                 "unsupported bundler version: #{Bundler::VERSION}. " \
                 "#{cic.class} does not respond to #info or #update_info"
         end
-        info = cic.instance_variable_get(:@cache).dependencies(name)
+        info = if defined?(::Bundler::CompactIndexClient::Parser)
+                 cic.info(name)
```

Comment:
do we maybe need to clear a cache here now, or something?

Reply:
I don't think so. We weren't clearing it before so I don't believe we need to clear it now?

Reply:
I think the caching is now at the level we are calling, whereas before we were calling into one level deeper

Reply:
I'm not sure I follow with my limited understanding of bundler internals I'm afraid 😞 
Happy to give it another look but I think I'll need a little nudge in the right direction